sanity-io/structured-content-2022
# Sponsorship information: remove `available` field

# Sponsorship information: remove `available` field

## Intent

Fixes [this](https://app.shortcut.com/sanity-io/story/17275/remove-of-available-sponsorships-from-sponsorship-tiers) Shortcut Story.

## Description

Figma not fully updated at the time of submitting this PR, margins set based on [Slack clarifications](https://wja.slack.com/archives/C02E2ERKR7E/p1649066453854369?thread_ts=1649066287.323309&cid=C02E2ERKR7E).

## Testing this PR

1. Visit [/sponsorship-information](https://structured-content-2022-web-git-sponsorships-delete-avai-7b5db6.sanity.build/sponsorship-information)
2. Verify that no `available` field is rendered
3. Verify that all references to the field is deleted from the _/web_ directory
